Resources/SeleniumApp.robot

```diff
@@ -1,14 +1,17 @@
 *** Settings ***
 Resource    PO/LandingPage/LandingPage.robot
+Resource    PO/SimpleForm/SimpleFormDemo.robot
```

Comment:
This should not be here because of the keyword on Simple Form Demo is interacting with LandingPage